Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -177,6 +177,17 @@ public static void animation(Session session, params string[] parameters)
             session.Network.EnqueueSend(new GameMessageMotion(session.Player, session, (MotionCommand)animationId, 1.0f));
         }
 
+        [CommandHandler("movement", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0)]
+        public static void Movement(Session session, params string[] parameters)
+        {
+            var movement = new MovementData { ForwardCommand = 24, MovementStateFlag = MovementStateFlag.ForwardCommand};
+
+            session.Network.EnqueueSend(new GameMessageMotion(session.Player, session, MotionActivity.Idle, MovementTypes.Invalid, MotionFlags.None, MotionStance.Standing,movement));

```

Comment:
1, you've got some inconsistent whitespace in this function.  2, can you document what this function does?  It's non-obvious to me.